core/lib/multistore.ts

```diff
@@ -30,21 +33,28 @@ function getExtendedStoreviewConfig (storeView: StoreView): StoreView {
   return storeView
 }
 
+/**
+ * Returns base storeView object that can be created without storeCode
+ */
+function buildBaseStoreView (): StoreView {
+  return cloneDeep({
+    tax: config.tax,
+    i18n: config.i18n,
+    elasticsearch: config.elasticsearch,
+    storeCode: null,
+    storeId: config.defaultStoreCode && config.defaultStoreCode !== '' ? config.storeViews[config.defaultStoreCode].storeId : 1,
+    seo: config.seo
+  })
+}
+
 export function currentStoreView (): StoreView {
-  // TODO: Change to getter all along our code
-  return rootStore.state.storeView
+  const serverStoreView = get(global, 'process.storeView', undefined)
```

Comment:
what about client-side? there is no global?

Reply:
there is no global on client-side, I could use window, but I think we can stay with same behavior as it was for client

Reply:
it is in next line   `const clientStoreView = get(rootStore, 'state.storeView', undefined)`